PRESS-RELEASE.md

```diff
@@ -1,44 +1,28 @@
-
-Oh, and I also like to write press-releases in what I call "Oprah-speak" for mainstream consumer products. Imagine you're sitting on Oprah's couch and have just explained the product to her, and then you listen as she explains it to her audience. That's "Oprah-speak", not "Geek-speak".
-
-Once the project moves into development, the press release can be used as a touchstone; a guiding light. The product team can ask themselves, "Are we building what is in the press release?" If they find they're spending time building things that aren't in the press release (overbuilding), they need to ask themselves why. This keeps product development focused on achieving the customer benefits and not building extraneous stuff that takes longer to build, takes resources to maintain, and doesn't provide real customer benefit (at least not enough to warrant inclusion in the press release).
- -->
  
 ## LANshark ##
-  <!-- > Name the product in a way the reader (i.e. your target customers) will understand. -->
+ > Working Title: Augmented History Tour
 
 ## Use Augmented Reality to discover facts about your city ##
-  <!-- > Describe who the market for the product is and what benefit they get. One sentence only underneath the title. -->
+  > This app is for history lovers and travelers who want to experience a new way of learning about the places they are in.
 
 ## Summary ##
-  > Allows people to use augmented reality on their mobile device to see points of interest their current area, and access that information by using their camera in real time to create an interactive cultural experience.
+  > Allows people to use augmented reality on their mobile device discover points of interest and facts related their current area, and, to access that information by using their smartPhones camera in real time to create an interactive cultural experience.
 
 ## Problem ##
-  > The traditional method of information delivery does not seamlessly display relevant information about points of interest, detracting from the overall user experience.
 
+  > Traditionally, discovering information about an area does not provide a seamless, interactive and immersive experience. The user is required to look up information via external means such as search engines, travel guides, etc which is distracting from the overall experience.
+  
 ## Solution ##
-  > Our app allows users to seamlessly learn about points of interest near them, though an interactive AR user experience.
+  > Our app allows users to seamlessly learn about points of interest near them,simple by pointing their smartphone at a structure, landmark or simply by being present in a location.
```

Comment:
Our app allows users to seamlessly learn about points of interest near them, **simply** by pointing their smartphone at a structure, landmark or simply by being present in a location.